apps/src/sites/studio/pages/levelbuilder_gamelab.js

```diff
@@ -62,29 +62,4 @@ $(document).ready(function() {
   if (document.getElementById('level_validation_code')) {
     initializeCodeMirror('level_validation_code', 'javascript');
   }
-  const autoRunSetup = document.getElementById('level_auto_run_setup');
-  const customSetupCode = document.getElementById('level_custom_setup_code');
```

Comment:
~Where did these elements get created?~ Never mind... they were removed in https://github.com/code-dot-org/code-dot-org/pull/29364.